.buildkite/scripts/build_docker_image.ps1

```diff
@@ -46,11 +46,19 @@ try {
     $version = "$($pathParts[-2])-$($pathParts[-1])"
     
 @"
+# escape=``
 FROM microsoft/windowsservercore
 MAINTAINER The Habitat Maintainers <humans@habitat.sh>
 ADD rootfs /
 WORKDIR /src
-RUN /hab/pkgs/$ident/bin/hab/hab.exe pkg exec core/windows-service install
+SHELL ["powershell", "-Command", "`$ErrorActionPreference = 'Stop'; `$ProgressPreference = 'SilentlyContinue';"]
+# Install the habitat windows service and then allow color and strip the log timestamps.
+# Because this is a studio we are not worried about ANSI codes in the log. Users will 
+# view them by tailing the log to the console. Because we are in a Docker studio, it is safe to
+# assume that the terminal supports ANSI sequence codes.
+RUN &/hab/pkgs/$ident/bin/hab/hab.exe pkg exec core/windows-service install; ``
+    (Get-Content /hab/svc/windows-service/HabService.dll.config).replace('--no-color', '') | Set-Content /hab/svc/windows-service/HabService.dll.config; ``
```

Comment:
Is this the only mechanism we have to affect the configuration? Is this expected to be the only scenario in which we'd want to do this?

Reply:
I think so. In any "normal" (ie non-studio) windows service install, we want the logs to be consumable in a non-console context. So it makes sense in all cases to make sure they have no ANSI codes and stripping the timestamps is nice to make the studio sup output "look" like normal supervisor output identical to a local studio or a linux studio.

The `windows-service` readme has instructions on editing the config and log4net.xml to satisfy a user's unique requirements but the defaults we have I think are good for most cases.